make_release.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -e
+set -f
+set -u
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <tag-name>" >&2
+    exit 1
+fi
+
+
+if ! [ $( git tag -l "$1" ) ]; then
+    echo "tag $1 not found" >&2
+    exit 1
+fi
+
+function create_demo_assets() {
+    cd demo/
+    make all clean
+
+    cd pdfpc-video-example
+    make all clean
+
+    zip -j ../pdfpc-video-example.zip video-example.pdf video-example.tex apollo17.avi apollo17.jpg
+    cd ../../
+}
+
+function create_release() {
+    API_URL=$1
+    TAG_NAME=$2
+
+    curl --data <<<EOF
```

Comment:
Shouldn't it be "<<" instead of "<<<"?

Reply:
You are right. But using heredoc at this point was wrong anyways :)